TWLight/users/admin.py

```diff
@@ -118,7 +118,7 @@ class UserAdmin(AuthUserAdmin):
     list_display = ["username", "get_wp_username", "is_staff"]
```

Comment:
```suggestion
    list_display = ["username", "get_wp_username", "email", "is_staff"]
```

Reply:
Hello @jsnshrmn, I added the email field in list_display. Thank you.